cupy/fallback_mode/fallback.py

```diff
@@ -0,0 +1,137 @@
+"""
+Main fallback class.
+"""
+import types
+import cupy as cp
+import numpy as np
+
+from cupy.fallback_mode.utils import FallbackUtil
+from cupy.fallback_mode.utils import call_cupy
+from cupy.fallback_mode.utils import call_numpy
+
+
+class RecursiveAttr(FallbackUtil):
+    """
+    RecursiveAttr class to catch all attributes corresponding to numpy,
+    when user calls fallback_mode. numpy is an instance of this class.
+    """
+    def __init__(self, name):
+        self.name = name
+
+    def notification_status(self):
```

Comment:
What is this method used for? What is notification status?

Reply:
It's for the notifications while a cupy method is falling back to numpy method. It's used [here].(https://github.com/cupy/cupy/pull/2229/files#diff-f4dc7ba9c20f6dfa57b5c42732dfe591R115)
Instead of using print statement, it could also be a warning. How about it?
Also just `notification` is better than `notification_status`. Should I change it?